Silicon/TigerlakePkg/FspBin/FspBin.inf

```diff
@@ -11,7 +11,7 @@
 
 [UserExtensions.SBL."CloneRepo"]
   REPO    = https://github.com/intel/FSP.git
-  COMMIT  = c4af5ac9f627571179879484d93093c31f426c96
+  COMMIT  = 851f7105d803672e17202bc2f598a5a4dbf659d3
 
```

Comment:
Do you need update TGL version (e.g.: VERINFO_PROJ_MINOR_VER) when updating FSP?

Reply:
for the IPU update, it's not a regular MR update so that no change on it.

Reply:
It is ok for me.